pay-api/migrations/versions/2024_07_22_f9c15c7f29f5_.py

```diff
@@ -0,0 +1,46 @@
+"""add primary key constraint to eft_refund_email_list table
+
+Revision ID: f9c15c7f29f5
+Revises: fb59bf68146d
+Create Date: 2024-07-22 14:52:01.050844
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.schema import Sequence, CreateSequence
+
+# revision identifiers, used by Alembic.
+revision = 'f9c15c7f29f5'
+down_revision = 'fb59bf68146d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_table('eft_refund_email_list')
```

Comment:
probably not the best practice, if someone was using this table? but should be fine

Reply:
this table is not deployed yet, so it's empty